sorc/chgres_cube.fd/search_util.f90

```diff
@@ -21,21 +17,23 @@ module search_util
 
  contains
 
- subroutine search (field, mask, idim, jdim, tile, field_num, latitude, terrain_land, soilt_climo)
-
 !-----------------------------------------------------------------------
-! Replace undefined values on the model grid with a valid value at
-! a nearby neighbor.  Undefined values are typically associated
-! with isolated islands where there is no source data.
-!
-! Routine searches a neighborhood with a radius of 100 grid points.
-! If no valid value is found, a default value is used.
+!> @brief Replace undefined surface values.
+!!
+!! Replace undefined values on the model grid with a valid value at
+!! a nearby neighbor.  Undefined values are typically associated
+!! with isolated islands where there is no source data.
+!!
+!! Routine searches a neighborhood with a radius of 100 grid points.
+!! If no valid value is found, a default value is used.
 !
-! Note: This routine works for one tile of a cubed sphere grid.  It
-! does not consider valid values at adjacent faces.  That is a 
-! future upgrade.
+!! Note: This routine works for one tile of a cubed sphere grid.  It
```

Comment:
You can change Note: to @note and Doxygen will format it as a note...

Reply:
Done.